curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/blur.html

```diff
@@ -0,0 +1,38 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>Hello OpenCV.js</title>
+<script async src="https://docs.opencv.org/master/opencv.js" onload="onOpenCvReady();" type="text/javascript"></script>
+<script type="text/javascript">
+
+  function onOpenCvReady() {
+    document.getElementById('status').innerHTML = 'OpenCV.js is load.';
+    cv["onRuntimeInitialized"]=()=> {
+       document.getElementById("run").disabled=false;
+  }
+  }
+  function runBlur(){
+    let src = cv.imread('imageSrc');
+    let dst = new cv.Mat();
+    let ksize = new cv.Size(3, 3);
```

Comment:
Please rename this variable to `kernelSize` (simpler to understand the meaning)

Reply:
done